test/math.jl

```diff
@@ -115,7 +115,7 @@ for (jl_func, seg_func) in [(sum, segment_sum), (prod, segment_prod), (minimum,
 end
 
 a = TensorFlow.constant(eye(5))
-idxs = TensorFlow.constant([1,1,2,3,1])
+idxs = TensorFlow.constant(map(Int64,[1,1,2,3,1]))
```

Comment:
Remove the `TensorFlow.` part of the the names -- `constant` is exported by `TensorFlow` module so not required here.

Also `map(Int64,[1,1,2,3,1])`  is equivalent to `Int64[1,1,2,3,1]`.
And that latter syntax is preferred.